Source/SIMPLib/DataContainers/DataContainer.h

```diff
@@ -73,8 +73,8 @@ class SIMPLib_EXPORT DataContainer : public Observable
   PYB11_METHOD(QString getInfoString ARGS InfoStringFormat)
   PYB11_METHOD(void addAttributeMatrix ARGS Name AttributeMatrix)
 
-  PYB11_METHOD(AttributeMatrix::Pointer getAttributeMatrix OVERLOAD const.QString.&, Name)
-  PYB11_METHOD(AttributeMatrix::Pointer getAttributeMatrix OVERLOAD const.DataArrayPath.&, Path)
+  PYB11_METHOD(AttributeMatrix::Pointer getAttributeMatrix OVERLOAD const.QString.&,Name)
+  PYB11_METHOD(AttributeMatrix::Pointer getAttributeMatrix OVERLOAD const.DataArrayPath.&,Path)
```

Comment:
Is the space after the comma important at the end of this macro?

Reply:
YES. The space MUST be there for our parsing to work.

Reply:
Is there a way to put a clang format exception for lines starting with PYB11_METHOD?

Reply:
Actually, no the space should NOT be there. I am working a better solution to this.